translations/README.bn.md

```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Star)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![English](https://img.shields.io/badge/-English-red)](README.md)
```

Comment:
Is the path to the English file correct?

I clicked on copy path and I got this

Reply:
Nope - this is in the `translations` folder, so you need to go up one level. Should be `../README.md`